Translator/Header/Writer.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#ifndef WRITER_H
```

Comment:
You can remove the #ifndef guards since #pragma once already prevents multiple inclusions